.gitlab-ci.yml

```diff
@@ -21,7 +21,17 @@ stages:
 # Test the SDPSubarray device
 subarray_device_tests:
   extends: .tango_tests
+  before_script:
+    - apt update
+    - apt install -y wget
+    - pip install ska-sdp-config
   script:
+    - cd /usr/local/src
+    - wget "https://github.com/coreos/etcd/releases/download/v3.3.9/etcd-v3.3.9-linux-amd64.tar.gz"
+    - tar -xvf etcd-v3.3.9-linux-amd64.tar.gz
+    - mv etcd-v3.3.9-linux-amd64/etcd* /usr/local/bin/
+    - etcd &
```

Comment:
See the solution on `config_db_tests`:
```
    services:
      - name: quay.io/coreos/etcd:latest
        alias: etcd
        command:
          - /usr/local/bin/etcd
          - "--advertise-client-urls=http://0.0.0.0:2379"
          - "--listen-client-urls=http://0.0.0.0:2379"
          - "--initial-advertise-peer-urls=http://0.0.0.0:2380"
          - "--listen-peer-urls=http://0.0.0.0:2380"
          - "--initial-cluster=default=http://0.0.0.0:2380"
    variables:
      SDP_TEST_HOST: etcd
```

It's a bit wordy mainly because etcd configuration is a bit all-or-nothing.